### DISCUSSION TITLE
if our dapp can be deployed properly in Apple App Store without being rejected by the Apple login provided by web3auth SDK??

### DISCUSSION THREAD

**Discussion by guyin123** [Web3Auth/Web3Auth#1269] [Help]
Hey, devteam~I have a question to confirm.
We are using plug and play SDK to integrate social login into our mobile dapp. According to the official Apple requirements we have to add the option of Apple login, but the Apple login through web3auth is in web form.
So we want to know if our dapp can be deployed properly in Apple App Store without being rejected by the Apple login provided by web3auth SDK?
Cause we don‘t know whether Apple official requires the native SDK. Could you give us some advice from your experience?

**Comment by YZhenY** [comment]
Some applications have managed to use the web login on iOS, but we've heard of some difficulties and pushback from Apple.

What i would recommend is using custom authentication with plug and play and hooking up the native Apple login via that which is supported. You might have to use a proxy like firebase inbetween.

**Reply by huben99** [reply]
If i want to use the native Apple login. then i only choose flutter and iOS SDK ?

**Reply by huben99** [reply]
if only use firebase, auth0 right ?
